edge-modules/TemperatureFilter/Program.cs

```diff
@@ -65,13 +65,13 @@ static async Task<int> MainAsync()
 
             Tuple<ModuleClient, ModuleConfig> userContext = moduleclientAndConfig;
 
-            await moduleclientAndConfig.Item1.SetInputMessageHandlerAsync("input1", PrintAndFilterMessages, userContext).ConfigureAwait(false);
+            await moduleclientAndConfig.Item1.SetInputMessageHandlerAsync("input1", PrintAndFilterMessages, userContext);
```

Comment:
can you help me understand the reason to remove "ConfigureAwait(false)" here?

Reply:
Primarily for consistency.. we don't use it anywhere in our code.